grafana-builder/grafana.libsonnet

```diff
@@ -399,23 +399,23 @@
     nullPointMode: 'null as zero',
     targets: [
       {
-        expr: 'histogram_quantile(0.99, sum(rate(%s_bucket%s[$__interval])) by (le)) * %s' % [metricName, selector, multiplier],
+        expr: 'histogram_quantile(0.99, sum(rate(%s_bucket%s[$__rate_interval])) by (le)) * %s' % [metricName, selector, multiplier],
```

Comment:
I've never used `histogram_quantile` so I dont know if this makes sense.. But I dont see why It wouldn't :man_shrugging: